_includes/tutorials/time-concepts/ksql/markup/answer/short-answer.adoc

```diff
@@ -0,0 +1,11 @@
+To process data based on *ingestion time*, also known as *log time*, which uses the Kafka record timestamp:
```

Comment:
```suggestion
To process data based on the Kafka record timestamp (either *create time* or *ingestion time*) :
```

Reply:
@bbejeck I took the spirit of this suggestion and applied it in https://github.com/confluentinc/kafka-tutorials/pull/647/commits/623bcbee8ce6af37b033bc0ca999c64775fdd4b2. WDYT?